todoman/cli.py

```diff
@@ -115,33 +149,23 @@ def command_wrap(*a, **kw):
               'remain stable regardless of configuration or version.')
 @click.pass_context
 @click.version_option(prog_name='todoman')
-def cli(ctx, color, porcelain):
+def cli(click_ctx, color, porcelain):
+    ctx = click_ctx.ensure_object(AppContext)
```

Comment:
It goes through the `ctx.obj` of all contexts and parent contexts, and sets the current `ctx.obj` to `AppContext()` if nothing is found.